packages/resources/src/Api.ts

```diff
@@ -53,7 +53,8 @@ export interface ApiProps {
   readonly accessLog?:
     | boolean
     | string
-    | apig.CfnStage.AccessLogSettingsProperty;
+    | apig.CfnStage.AccessLogSettingsProperty
+    | apigV2AccessLog.AccessLogProps;
```

Comment:
Yes it is. Updated.

Reply:
Only is `retention` is optional otherwise old `apig.CfnStage.AccessLogSettingsProperty` will produce an error like this one:
```
Type '{ format: string; }' is not assignable to type 'string | boolean | AccessLogProps'.
  Property 'retention' is missing in type '{ format: string; }' but required in type 'AccessLogProps'.
```
I made retention optional.